packages/react-core/src/components/ToggleGroup/examples/ToggleGroupDefaultSingle.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+
+export const ToggleGroupDefaultSingle: React.FunctionComponent = () => {
+  const [isSelected, setIsSelected] = React.useState('');
+  const handleItemClick = (isSelected, event) => {
+    const id = event.currentTarget.id;
+    setIsSelected(id);
+  };
+  return (
+    <ToggleGroup aria-label="Default with single selectable">
+      <ToggleGroupItem
+        text="Option 1"
+        buttonId="firstRadio"
+        isSelected={isSelected === 'firstRadio'}
```

Comment:
For the buttonId (would apply to the other 2 ToggleGroupItem's in this example):

```suggestion
        buttonId="toggle-group-single-1"
        isSelected={isSelected === 'toggle-group-single-1'}
```